internal/kogitoinfra.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"github.com/kiegroup/kogito-operator/api"
+	"github.com/kiegroup/kogito-operator/core/client/kubernetes"
+	"github.com/kiegroup/kogito-operator/core/manager"
+	"github.com/kiegroup/kogito-operator/core/operator"
+	v1 "github.com/kiegroup/rhpam-kogito-operator/api/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// NewKogitoInfraHandler ...
+func NewKogitoInfraHandler(context operator.Context) manager.KogitoInfraHandler {
+	return &kogitoInfraHandler{context}
+}
+
+// NewNoOpKogitoInfraHandler returns a new instance of manager.KogitoInfraHandler which won't try to interact with the api.KogitoInfraInterface object
+func NewNoOpKogitoInfraHandler(context operator.Context) manager.KogitoInfraHandler {
```

Comment:
kogitoInfraHandlerNoOp seems unused, can it be removed?

Reply:
yes, its unused. I had removed it.